### PR TITLE
wizard performance enabled

### PR DESCRIPTION
**Pull request by bruno-garcia**
@lucas-zimerman @jennmueng Would you please help review the relevant wizards?

### CODE REVIEW

**Review by silent1mezzo — APPROVED** [review]

File: src/wizard/capacitor/index.md [hunk 1]
```diff
@@ -80,14 +80,26 @@ With Ionic/Angular:
 // app.module.ts
 import * as Sentry from "@sentry/capacitor";
 import * as SentryAngular from "@sentry/angular";
+// If taking advantage of automatic instrumentation (highly recommended)
+import { Integrations as TracingIntegrations } from "@sentry/tracing";
+// Or, if only manually tracing
+// import * as _ from "@sentry/tracing"
```
Comment (jennmueng) [inline src/wizard/capacitor/index.md:86]:
```suggestion
// import "@sentry/tracing";
```

**Review by jennmueng — APPROVED** [review]

File: src/wizard/react-native/index.md [hunk 2]
```diff
@@ -36,18 +36,33 @@ The call to the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will
...
 });
 ```
 
 The `sentry-wizard` will try to add it to your `App.js`
 
+### Wrap your app
+
+Wrap your app with Sentry to automatically instrument it with [touch event tracking](/platforms/react-native/touchevents/) and [automatic performance monitoring](/platforms/react-native/performance/instrumentation/automatic-instrumentation/):
+
+```javascript {filename:App.js}
```
Comment (jennmueng) [inline src/wizard/react-native/index.md:58]:
```suggestion
```javascript
```

File: src/wizard/android/index.md [hunk 3]
```diff
@@ -40,6 +40,9 @@ Add your DSN to the manifest file.
 ```xml {filename:AndroidManifest.xml}
 <application>
     <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
+    <!-- Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+       We recommend adjusting this value in production. -->
+    <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
```
Comment (marandaneto) [inline src/wizard/android/index.md:45]: there's already a section down this page, called Performance Monitoring, should we repeat it?

File: src/wizard/java/index.md [hunk 4]
```diff
@@ -51,7 +51,7 @@ import io.sentry.Sentry;
 
 Sentry.init(options -> {
   options.setDsn("___PUBLIC_DSN___");
-  // Set traces_sample_rate to 1.0 to capture 100% of transactions for performance monitoring.
+  // Set TracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
```
Comment (marandaneto) [inline src/wizard/java/index.md:54]:
```suggestion
  // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
```

File: src/wizard/react-native/index.md [hunk 5]
```diff
@@ -36,18 +36,33 @@ The call to the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will
...
+### Initialize the SDK
 
 ```javascript
 import * as Sentry from "@sentry/react-native";
 
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
+  // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+  // We recommend adjusting this value in production.
+  tracesSampleRate: 1.0
```
Comment (marandaneto) [inline src/wizard/react-native/index.md:48]:
```suggestion
  tracesSampleRate: 1.0,
```

File: src/wizard/android/index.md (hunk 3 above)
Comment (marandaneto) [reply src/wizard/android/index.md:45]: we can link down this page to the automatic instrumentation that requires adding an integration, similar to RN

Comment (bruno-garcia) [reply src/wizard/android/index.md:45]: Discussed on a call but to have it here: We want to experiment having this opt-in on the first copy-paste users do.

**Pull request merged**